### ISSUE TITLE
How can I make my "mainframe" persistent?

### ISSUE THREAD

**Issue by lemmy04** [RattyDAVE/docker-ubuntu-hercules-mvs#4]
Right now, if I make any changes inside MVS (create a user account etc), it's all lost if I recreate the container.
To make changes persistent I'd normally mount one or more volumes inside the container, but how many, and where?
Do I just "relocate" the whole /opt/hercules/tk4/dasd into a volume?
Anything else that I need to store?

**Comment by RattyDAVE** [comment]
Ahhh yes... I have been thinking about that.... And in the best solution is NOT to update the container.

If you want something more permanent then a full install is the best solution. From http://wotho.ethz.ch/tk4-/ get the tk4-_v1.00_current.zip

You COULD relocate the volumes and also the conf files. You could then "save" the files. Also you could run in another local Hercules instance.

TL;DR 
Yes. But is it worth the effort.

Dave
